percona-backup-mongodb/Dockerfile

```diff
@@ -14,6 +14,8 @@ LABEL org.opencontainers.image.title="Percona Backup for MongoDB" \
     low-impact solution for achieving consistent backups of MongoDB Sharded Clusters and Replica Sets." \
     org.opencontainers.image.authors="info@percona.com"
 
+LABEL maintainer="Percona Development <info@percona.com>"
```

Comment:
we can move this label into https://github.com/percona/percona-docker/pull/1112/files#diff-a0eb8e00613d547a6f2cf9d48780ed62709d5fafc206fc4fa9a40f7b47eb0b89R4 section